packages/core/admin/admin/src/pages/Home/HomePage.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
```

Comment:
this line is useless, you don't need to import React

Reply:
will remove it in my next PR thanks